package.json

```diff
@@ -67,5 +81,21 @@
       "no-duplicate-imports": "off",
       "@typescript-eslint/consistent-type-imports": "error"
     }
+  },
+  "nyc": {
+    "extends": "@istanbuljs/nyc-config-typescript",
```

Comment:
There is a nycrc file under @itwin/build-tools/.nycrc, so set the `nycrc-path` to the path which leads to that file. Then you will no longer need to extend that external `@istambuljs` package and you will be able to remove it from dependency list.

Reply:
added nycrc path